anti.md

```diff
@@ -58,12 +93,24 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Very little of their time would be spent on language design because writing in
+the language is the smallest part of the user experience. A ton of time would
+be necessary to parse websites and figure out how to get all of their different
+types of data and updates together.
```

Comment:
True, true.  A big reason why a DSL isn't a good solution / doesn't really solve the problem.